source/pmc/community-growth.md

```diff
@@ -3,90 +3,90 @@ title: Community Growth
-documentation is super clear and easy to follow.  Long-time committers
-on a project often forget all the institutional knowledge they just
-"know", so ensuring the "getting started" document actually works
-for newcomers is always worth looking at.
+Ensure your contribution documentation -- covering how to contribute,
+build, test, and submit pull requests -- is clear and easy to follow.
+Experienced committers often overlook the institutional knowledge
+they take for granted, so regularly review your "getting started"
+documentation from a newcomer's perspective.
 
-Encourage newcomers to talk about, and document, their pain points
-during their first contributions, and work towards fixing, or at least
-documenting, those things.
+Encourage new contributors to share and document their pain points
+during their first contributions. Work toward addressing or at least
+documenting these challenges to improve the experience for future
+contributors.
+
+Do not neglect non-code contributions as you think about where people
+can get involved.
 
 ## Publish your contributor ladder
 
-A [contributor ladder](/contributor-ladder.html) is a description of
-various levels of access and responsibility a contributor can progress
-through in your project. This is typically the path from contributor, to
-committer, to PMC member.
+A [contributor ladder](/contributor-ladder.html) describes the various
+levels of access and responsibility contributors can progress through
+in your project -- typically from contributor to committer to PMC member.
 
-Explicitly publish your requirements for each step. For example, if you
-require ten non-trivial patches accepted, or perhaps 3 months of
-involvement, document this on your website so that people don't feel
-that it's random, or that they are driving in the dark.
+Clearly document requirements for each level. For example, if you require
+ten substantial patches or three months of involvement, publish these
+criteria on your website. This prevents contributors from feeling the
+process is arbitrary or unclear.
 
-Consider lowering your bar to entry. Remember that while the risk of
-inviting someone "too early" is that they'll break something, that's why
-you have version control - so you can revert those changes. However, the
-risk of inviting someone too late is that they'll give up and go away
-forever.
+Consider lowering your entry barriers. Remember that inviting someone
+"too early" risks temporary issues that version control can fix, but
+inviting someone too late risks losing them permanently.
 
 ## Engage with big users
 
-If you are aware of companies that rely on your project, encourage them
-to think about what they would do if the project retired. This often
-leads to those companies realizing the value they get "for free", and
-starting to contribute in some way.
+Reach out to companies that depend on your project and ask them to consider
```

Comment:
To me that's close to FUD, "scare them to get their help", maybe there's a more positive way to say the same thing?

> Make sure companies that rely on your project realize its importance and are aware of how they can ensure it stays around for a long time. Encourage them to contribute instead of just consuming your software.

Reply:
That's fair. I'll attempt to rephrase that.